src/lsm/manifest_log.zig

```diff
@@ -536,10 +540,14 @@ pub fn ManifestLogType(comptime Storage: type) type {
                 .update,
                 => {
                     const extent = manifest_log.table_extents.getOrPutAssumeCapacity(table.address);
-                    if (extent.found_existing) {
-                        maybe(table.label.event == .insert); // (Compaction.)
-                    } else {
+                    if (!extent.found_existing) {
```

Comment:
Swapped the `if` here to line up with the `getOrPutAssumeCapacity` in `open_read_block_callback`.